fineract-provider/src/main/java/org/apache/fineract/infrastructure/campaigns/sms/service/SmsCampaignDropdownReadPlatformServiceImpl.java

```diff
@@ -78,8 +78,9 @@ public SmsCampaignDropdownReadPlatformServiceImpl(final SmsConfigUtils smsConfig
             ResponseEntity<Collection<SmsProviderData>> responseOne = restTemplate.exchange(uri, HttpMethod.GET, entity,
                     new ParameterizedTypeReference<Collection<SmsProviderData>>() {});
             smsProviderOptions = responseOne.getBody();
-            if (!responseOne.getStatusCode().equals(HttpStatus.OK)) {
-            }
+            //Better approch? Maybe log/throw exception that this happen?
+            // if (!responseOne.getStatusCode().equals(HttpStatus.OK)) {
+            // }
         } catch (Exception e) {
         }
```

Comment:
while you are already here anyway, let's get rid of this overly broad ` } catch (Exception e) {` as well? see if you can just remove it, or if not, make it catch a more specific exception, and then re-throw that. No need to log it if you rethrow. As per https://github.com/apache/fineract#error-handling-guidelines

Reply:
this was done